virttest/shared/cfg/base.cfg

```diff
@@ -1082,3 +1082,15 @@ uuid_dimm = ""
 #vm_sgx_epc_memdev_sgx0 = epc0
 #vm_sgx_epc_node_sgx0 = 0
 # Note: # if also enable guest numa node with sgx, please also add numa params
+#
+#
+# 
+# 
+# Both qemu-img create and convert can be operated with extent_size_hint
+#its default value is 1M. The avaliable value set range is [1M, 3.5G].
+# Set image_extent_size_hint
+#image_extent_size_hint = 0 #No extent_size_hint displayed in qemu-img info
```

Comment:
The value is `0`, at this point I am a bit confused about it.
Does it disable the functionality  `extent_size_hint` by setting `0`? or others?
And What happened from the perspective of `qemu-img` if we set the value out of the `[1M, 3.5G]` ?
Thanks.

Reply:
@YongxueHong, yes, when we set extent_size_hint=0, we disable it, and no extent_size_hint displayed in qemu-img info.

@YongxueHong, seting the value out of range [1M, 3.5G] is a negative test. I'm not sure we need to explain it here.

Reply:
IMO, 0 is also a valid value used for disabling the extent_size_hint. So we enable it by setting a value in range [1M, 3.5G], while we disable it by setting 0. CC @aliang123 @YongxueHong

Reply:
> Hi @aliang123 @zhencliu I got your point. So from my understanding, the qemu-img will take different actions according to the value passed, right?
> 
> If it is, I suggest that don't explain them again to avoid some misunderstandings. Since the user should know the meaning of the different values for the `extent_size_hint`. We just explain it relates to one of the options of `qemu-img` here.
It's OK for me, thanks. 

> 
> BTW, if the `qemu-img` product has some changes in the future, such as the valid value scope` [1M 3.5M]` is not available then we have to update this comment section.